frontend/wallet/src/render/components/Modal.re

```diff
@@ -43,20 +55,27 @@ module Binding = {
       ~contentLabel: string,
       ~className: string=?,
       ~overlayClassName: string=?,
+      ~appElement: Dom.element,
       ~children: React.element
     ) =>
     React.element =
     "react-modal";
+
+  type document;
+  [@bs.val] external document: document = "";
+  [@bs.send] external getElementById: (document, string) => Dom.element = "";
 };
 
 // Wrap react-modal with a default style
 [@react.component]
-let make = (~isOpen, ~onRequestClose, ~contentLabel, ~children) =>
+let make = (~isOpen, ~isRed=false, ~onRequestClose, ~title, ~children) =>
   <Binding
     isOpen
     onRequestClose
-    contentLabel
+    contentLabel=title
+    appElement={Binding.getElementById(Binding.document, "index")}
```

Comment:
You could probably grab this in a component did mount hook instead of every rerender but this is probably fine.